src/app/app.component.html

```diff
@@ -1,21 +1,51 @@
 <!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
-  <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
+<!--<div style="text-align:center">-->
+<mat-toolbar id="SummerDeeds">
+    <div fxShow="true" fxHide.gt-sm="true">
+        <button mat-icon-button (click)="sidenav.toggle()">
+            <mat-icon>menu</mat-icon>
+        </button>
+    </div>
+    <div class="navlinks">
+        <a mat-button class="companyName" routerLink="/">
+            <span class="summer">summer</span>
+            <span class="GDP"> GDP</span>
+        </a>
+    </div>
+    <img class="square" src="assets/square.png" alt="" />
+    <span class="example-spacer"></span>
+    <div class="navlinks">
+        <div fxShow="true" fxHide.lt-md="true" fxLayoutGap="20px">
+            <a class="navtext" mat-button routerLink="home">Home</a>
+            <a mat-button routerLink="jobs">Jobs</a>
+            <a *ngIf="user()" mat-button routerLink="create-job">New job</a>
+            <a *ngIf="!user()" mat-button routerLink="register">Register</a>
+            <a *ngIf="!user()" mat-button routerLink="login">Login</a>
+            <a *ngIf="user()" mat-button routerLink="logout">Logout</a>
+        </div>
+    </div>
+</mat-toolbar>
+<mat-sidenav-container fxFlexFill>
+    <mat-sidenav #sidenav fxLayout="column" mode="over" opened="false" fxHide.gt-sm="true">
+        <div fxLayout="column">
+            <a mat-button routerLink="home">Home</a>
+            <a mat-button routerLink="jobs">Jobs</a>
+            <a *ngIf="user()" mat-button routerLink="create-job">New job</a>
```

Comment:
try to avoid calling methods in *ngIf's cuz it's just calling it all the time to check if anything has changed and it's not efficient